test/fixtures/discovery/cluster/sbd/invalid_sbd_config

```diff
@@ -0,0 +1,3 @@
+SBD_DEVICE: /dev/vdc;/dev/vdb
```

Comment:
Could we maybe rename the file to `sbd_config_invalid`. 
All the other fixture files start with this prefix